### PR TITLE
modify integration file to take client id and secret from db

### PR DESCRIPTION
**Pull request by novisshrivv**

### CODE REVIEW

File: authlib/integrations/django_client/integration.py [hunk 1]
```diff
@@ -39,7 +40,11 @@ def generate_access_token_params(self, request_token_url, request):
 
     @staticmethod
     def load_config(oauth, name, params):
-        config = getattr(settings, 'AUTHLIB_OAUTH_CLIENTS', None)
+        auth_client = OauthClient.objects.get(name=name)
+        config = {}
+        config[auth_client.name] = {}
+        config[auth_client.name]['client_id'] = auth_client.oauth_client_id
+        config[auth_client.name]['client_secret'] = auth_client.oauth_client_secret
```
Comment (HimanshuS24) [inline authlib/integrations/django_client/integration.py]:
```
config = {
	auth_client.name: {
		'client_id': auth_client.oauth_client_id,
		'client_secret': auth_client.oauth_client_secret,
                 .........
	}
}
```

Comment (novisshrivv) [reply authlib/integrations/django_client/integration.py]: added changes

**Review by HimanshuS24 — DISMISSED** [review]

**Comment by HimanshuS24** [comment]
@shivinquartic @manoj016 can we add a minimum of two reviewers rule to this repo as we have in others repo in our platform?

File: authlib/integrations/django_client/integration.py [hunk 2]
```diff
@@ -39,7 +40,13 @@ def generate_access_token_params(self, request_token_url, request):
 
     @staticmethod
     def load_config(oauth, name, params):
```
Comment (shivinquartic) [inline authlib/integrations/django_client/integration.py:42]: comments

**Comment by shivinquartic** [comment]
@novis-opscale please raise this against develop.

File: authlib/integrations/django_client/integration.py (hunk 2 above)
Comment (novisshrivv) [reply authlib/integrations/django_client/integration.py:42]: added

File: authlib/consts.py [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
 author = 'Quartic.ai Engineering Team'
 homepage = 'https://github.com/Quarticai/authlib'
+
+default_user_agent = '{}/{} (+{})'.format(name, version, homepage)
+
+default_json_headers = [
+    ('Content-Type', 'application/json'),
+    ('Cache-Control', 'no-store'),
+    ('Pragma', 'no-cache'),
+]
```
Comment (shivinquartic) [inline authlib/consts.py:12]: wha are these changes for?

Comment (novisshrivv) [reply authlib/consts.py:12]: default user agent and default json headers are being used inside the integration class of base client in master branch. But these lines were removed in develop branch and was throwing error so I added them back

Comment (shivinquartic) [reply authlib/consts.py:12]: ohh okay 👍

**Review by shivinquartic — APPROVED** [review]

**Review by abhishekkalhan — APPROVED** [review]

**Pull request merged**